src/main/java/mate/academy/spring/dao/impl/MovieSessionDaoImpl.java

```diff
@@ -57,4 +58,49 @@ public Optional<MovieSession> get(Long id) {
             throw new DataProcessingException("Can't get a movie session by id: " + id, e);
         }
     }
+
+    @Override
+    public void delete(Long id) {
+        Session session = null;
+        Transaction transaction = null;
+        MovieSession sessionById = get(id).orElseThrow();
```

Comment:
```suggestion
    public void delete( MovieSession  movieSession) {
        Session session = null;
        Transaction transaction = null;
```
pass in on controller layer